tests/create-rapid/enc/enc.cpp

```diff
@@ -13,7 +13,7 @@ OE_SET_ENCLAVE_SGX(
     1,    /* ProductID */
     1,    /* SecurityVersion */
     true, /* Debug */
-    8,    /* NumHeapPages */
+    128,  /* NumHeapPages */
```

Comment:
Although this isn't threaded, snmalloc's class size scheme means slightly higher memory usage in this case than with dlmalloc.